### ISSUE TITLE
Кредит 

### ISSUE THREAD

**Issue by Davibiyuu** [Davibiyuu/zashitacoil#253]
Реально ли взять кредит с не очень хорошей кредитной историей?

**Comment by diorjio** [comment]
Сейчас есть немало разнообразных предложений, вопрос лишь в том, какие процентные ставки там будут. Лично у меня, как и у многих, испорчена кредитная история, поэтому путь к максимально выгодным процентным ставкам для меня был закрыт. По крайней мере мне так казалось. Когда срочно понадобилась крупная сумма денег и два банка отказали в кредите, я поняла, что нужно обращаться к профессионалам. В итоге, [кредитный брокер](https://www.kp.ru/daily/27591/4941693/?ysclid=lybn2eg0kk768141951&erid=LjN8K3y2a) оказался настоящим спасением. В статье прочла про Кредит Консалтинг, это компания, у которой одобряемость кредитов до 95%, и они работают с любой кредитной историей. С их помощью я смогла получить кредит на сумму, необходимую для моего бизнеса, и при этом процентная ставка была всего 4.5%. Кроме того, они помогли мне отменить страховку, что сэкономило до 25% от общей суммы кредита. Что меня особенно поразило, так это их оперативность. Получение денег у них в офисе заняло всего 15-120 минут. Они сотрудничают с около 50 банками-партнерами, включая такие крупные, как Альфа-Банк, Сбербанк и Совкомбанк, а также помогают получать займы от частных инвесторов. Работают на финансовом рынке с 2005 года, что внушает доверие. Для тех, кто находится в сложной финансовой ситуации и имеет плохую кредитную историю, я рекомендую обратить внимание на эту компанию

**Comment by CatrinStaff** [comment]
Да, многие банки одобряют кредиты даже с плохой кредитной историей. Вы можете взять кредит 2 млн [https://www.otpbank.ru/retail/credits/cash/2000000-rublej/](https://www.otpbank.ru/retail/credits/cash/2000000-rublej/) . Просто я сам брал средства в банке недавно и у меня не очень хорошая КИ. В ОТП банке заодно ознакомитесь со ставками и попробуйте оставить заявку. Там быстро принимают решение. В любом случае, это хороший банк и условия вы там получите выгодные. Удачи!
